Dockerfile.ppc64le

```diff
@@ -12,37 +29,82 @@ FROM ppc64le/gcc:5.3
 RUN apt-get update && apt-get install -y \
 	apparmor \
 	aufs-tools \
+	automake \
+	bash-completion \
 	btrfs-tools \
 	build-essential \
+	createrepo \
 	curl \
+	dpkg-sig \
 	git \
 	iptables \
 	jq \
-	net-tools \
```

Comment:
We still need this to get ifconfig for the daemon integration tests. Other than that everything looks good on power.